File: BeaverTinder.Shared/Dto/Subscription/SubscriptionInfoDto.cs
namespace BeaverTinder.Shared.Dto.Subscription;

public class SubscriptionInfoDto
{
    public int Id { get; set; }
    public string Name { get; set; } = default!;
    public double PricePerMonth { get; set; }
    public string Description { get; set; }
}

File: mobile/lib/dto/subscription/subscription_info_dto.dart
class SubscriptionInfoDto {
  final int id;
  final String name;
  final int pricePerMonth;
  final String description;


  SubscriptionInfoDto(
      this.id,
      this.name,
      this.pricePerMonth,
      this.description);


  factory SubscriptionInfoDto.fromJson(dynamic json)
  {
      return SubscriptionInfoDto(
        json["id"],
        json["name"],
        json["pricePerMonth"],
        json["description"]
      );
  }
}